### ISSUE TITLE
App crashes then spits error when deleting multiple files

### ISSUE THREAD

**Issue by cu1tur4l** [SimpleMobileTools/Simple-File-Manager#632]
Hello, 

tl;dr 
App crashes when deleting multiple files, sometimes even one file. Spits out this toast error: "Error: java.lang.illegalArgumentException: Volume external_primary not found". Files are still there, if I repeat this the app usually will not crash and will delete one or two files. Repeat again and so on and so on. It is faster to delete files one by one since then there is (usually) no error or crash.

App version: 6.14.0 from F-Droid
Android 12, with Vivo FunTouchOS 12
Vivo V21 5G 

Probably not needed, but "more info": 
This problem started about 3(?) months ago, but I'm not deleting things that often so I didn't do anything about it. Since it was that long ago, I'm not sure if what I'm saying is correct, but I think that's how it was; it all started after FunTouchOS 12 (Vivo's phone OS) update which also updated the android version to android 12. Simple File Manager and Simple Gallery started giving this - "Error: java.lang.illegalArgumentException: Volume external_primary not found" after deleting multiple files, or sometimes even one file. If it does crash, I go back to the app it shows me the toast error, and select the files again, then it spits out the error again but this time usually does not crash and deletes one or two files, leaving rest untouched but until refreshing the files aren't there.

Any ideas :)?

**Comment by tibbi** [comment]
try reinstalling the app. And report just 1 thing here at once, do not mix multiple things, multiple apps not for sure.

**Comment by cu1tur4l** [comment]
Sorry about that. I deleted the Simple Gallery thing from the post.
About reinstalling; I think I've tried it already. Will double check now.

**Comment by cu1tur4l** [comment]
Still crashes. But now that I had to create files to delete, I noticed that it didn't crash when deleting .jpg and did crash when deleting .pdf. Double checked it.
